### PR TITLE
reinforcement_learning folder updates

### PR DESCRIPTION
**Pull request by goelakash**
*Issue #, if available:*

*Description of changes:*
1. Update tensorflow imports in `tf_serving_utils.py` 
2. Update `rl_cartpole_ray` to use RL Toolkit and RL Framework params.
3. Add/update `job_name` parameter for several notebooks 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: reinforcement_learning/rl_cartpole_ray/rl_cartpole_ray_gymEnv.ipynb [hunk 1]
```diff
@@ -178,33 +178,6 @@
     "    !/bin/bash ./common/setup.sh"
```
Comment (leijin168) [inline reinforcement_learning/rl_cartpole_ray/rl_cartpole_ray_gymEnv.ipynb:178]: <p>why we remove the 'Use docker image section'? We can say we are using ray version 0.8.5.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1877/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

File: reinforcement_learning/rl_hvac_coach_energyplus/rl_hvac_coach_energyplus.ipynb [hunk 2]
```diff
@@ -459,7 +459,7 @@
     "                             instance_type=instance_type,\n",
```
Comment (leijin168) [inline reinforcement_learning/rl_hvac_coach_energyplus/rl_hvac_coach_energyplus.ipynb:459]: <p>Do all of our RL notebooks support python 3.7 or above?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1877/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

File: reinforcement_learning/rl_portfolio_management_coach_customEnv/rl_portfolio_management_coach_customEnv.ipynb [hunk 3]
```diff
@@ -557,6 +557,7 @@
     "                      entry_point=\"evaluate-coach.py\",\n",
```
Comment (leijin168) [inline reinforcement_learning/rl_portfolio_management_coach_customEnv/rl_portfolio_management_coach_customEnv.ipynb:557]: <p>Looks like this notebook is really old (coach 0.11.0) and probably should be one of the first batch to convert to Ray.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1877/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

File: reinforcement_learning/rl_cartpole_ray/rl_cartpole_ray_gymEnv.ipynb (hunk 1 above)
Comment (goelakash) [reply reinforcement_learning/rl_cartpole_ray/rl_cartpole_ray_gymEnv.ipynb:178]: We don't mention this in the notebooks where we use the ray-version param in the `RLEstimator` function. Just following that convention here.

File: reinforcement_learning/rl_hvac_coach_energyplus/rl_hvac_coach_energyplus.ipynb (hunk 2 above)
Comment (goelakash) [reply reinforcement_learning/rl_hvac_coach_energyplus/rl_hvac_coach_energyplus.ipynb:459]: They should, we don't use any deprecated Python 3 functions.

File: reinforcement_learning/rl_hvac_coach_energyplus/rl_hvac_coach_energyplus.ipynb [hunk 4]
```diff
@@ -489,9 +489,16 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "predictor = estimator.deploy(initial_instance_count=1,\n",
```
Comment (yijiezh) [inline reinforcement_learning/rl_hvac_coach_energyplus/rl_hvac_coach_energyplus.ipynb:492]: Any reason for this change? Does MXNet 1.8.0 compatible with what exported from Ray

Comment (goelakash) [reply reinforcement_learning/rl_hvac_coach_energyplus/rl_hvac_coach_energyplus.ipynb:492]: Good question, I did not see the hosting succeed with the generic `estimator.deploy` function, which is why I replaced that with a `MXNetModel` instead. The `1.8.0` version is the latest and the notebook didn't throw any errors, so I'm assuming its backwards compatible with the MXNet version (1.3.0) of the sagemaker-rl coach image for hosting the model.

Comment (yijiezh) [reply reinforcement_learning/rl_hvac_coach_energyplus/rl_hvac_coach_energyplus.ipynb:492]: Others LGTM

**Review by yijiezh — APPROVED** [review]

**Comment by goelakash** [comment]
Closing this in favor of other small PRs

**Pull request closed without merging**